doc-content/enterprise-only/getting-started-kogito/proc-kogito-microservices-deploying-on-ocp-console.adoc

```diff
@@ -0,0 +1,100 @@
++
+.SmallRye Heath dependency for Quarkus applications on OpenShift
+[source,xml]
+----
+<dependency>
+  <groupId>io.quarkus</groupId>
+  <artifactId>quarkus-smallrye-health</artifactId>
+</dependency>
+----
+
+.Procedure
+. In the OpenShift web console, go to *Operators* -> *OperatorHub* in the left menu, search for and select *Kogito*, and follow the on-screen instructions to install the latest operator version.
+. After you install the {KOGITO} Operator, go to *Operators* -> *Installed Operators* and select *Kogito*.
+. In the operator page, select the *Kogito Build* tab and click *Create KogitoBuild* to create the {KOGITO} build definition.
+. In the application window, use the *Form View* or *YAML View* to configure the build definition.
++
+--
+At a minimum, define the application configurations shown in the following example YAML file:
+
+.Example YAML definition for an application with {KOGITO} build
+[source,yaml,subs="attributes+"]
+----
+apiVersion: app.kiegroup.org/v1beta1 # {KOGITO} API for this service
+kind: KogitoBuild  # Application type
+metadata:
+  name: example-quarkus  # Application name
+  namespace: kogito  # OpenShift project namespace
+spec:
+  type: RemoteSource
+  gitSource:
+    uri: 'https://github.com/kiegroup/kogito-examples'  # Git repository containing application (uses default branch)
+    contextDir: process-quarkus-example  # Git folder location of application
+----
+
+[NOTE]
+====
+If you configured an internal Maven repository, you can use it as a Maven mirror service and specify the Maven mirror URL in your {KOGITO} build definition to substantially shorten build time:
+
+[source,yaml]
+----
+spec:
+  mavenMirrorURL: http://nexus3-nexus.apps-crc.testing/repository/maven-public/
+----
+
+For more information about internal Maven repositories, see the https://maven.apache.org/guides/introduction/introduction-to-repositories.html[Apache Maven] documentation.
+====
+--
+. After you define your application data, click *Create* to generate the {KOGITO} build.
++
+Your application is listed in the *{KOGITO}Builds* page. You can select the application name to view or modify application settings and YAML details.
+
+. In the operator page, select the *Kogito Service* tab and click *Create KogitoRuntime* to create the {KOGITO} microservice definition.
+. In the application window, use the *Form View* or *YAML View* to configure the microservice definition.
++
+--
+At a minimum, define the application configurations shown in the following example YAML file:
+
+.Example YAML definition for an application with {KOGITO} microservices
+[source,yaml,subs="attributes+"]
+----
+apiVersion: app.kiegroup.org/v1beta1 # {KOGITO} API for this microservice
```

Comment:
for RMPAM apiversion should be rhpam.kiegroup.org/v1

Reply:
some screenshot could be helpful in process to deploy Kogito application using openshit web console.